test.ipynb

```diff
@@ -232,7 +232,7 @@
   },
```

Comment:
_Commented on notebook `test.ipynb` Cell 5 Line 2_
```python
sns.histplot(x='total_bill', data=data, kde=True, hue='sex')

```
How can we trust the samples???????

_Respond and view the context [here](https://gitnotebooks.com/Sara-mibo/Test-review-notebooks/pull/1)._